ClientLib/src/main/java/com/o3dr/services/android/lib/coordinate/LatLongAlt.java

```diff
@@ -14,24 +15,34 @@
      * Stores the altitude in meters.
      */
     private double mAltitude;
+    private Frame mFrame;
+
+    public LatLongAlt() {
+        super();
+        mAltitude = 0.0;
+        mFrame = Frame.GLOBAL_RELATIVE;
+    }
 
-    public LatLongAlt(double latitude, double longitude, double altitude) {
+    public LatLongAlt(double latitude, double longitude, double altitude, Frame frame) {
         super(latitude, longitude);
         mAltitude = altitude;
+        mFrame = frame;
     }
 
-    public LatLongAlt(LatLong location, double altitude){
+    public LatLongAlt(LatLong location, double altitude, Frame frame){
         super(location);
         mAltitude = altitude;
+        mFrame = frame;
     }
 
     public LatLongAlt(LatLongAlt copy) {
-        this(copy.getLatitude(), copy.getLongitude(), copy.getAltitude());
+        this(copy.getLatitude(), copy.getLongitude(), copy.getAltitude(), copy.getFrame());
     }
 
-    public void set(LatLongAlt source){
+    public void set(LatLongAlt source){ // TODO: this looks wrong
```

Comment:
It's like a shallow copy, i'm not sure when set is called and why? why is using set different from 
latLgAlt1 = latLgAlt2